cluster-operator/src/test/java/io/strimzi/operator/cluster/model/ListenersUtilsTest.java

```diff
@@ -761,4 +762,19 @@ public void testGetExternalAdvertisedUrlWithoutOverrides() {
         assertThat(ListenersUtils.advertisedPortFromOverrideOrParameter(listener, 0, 12345), is("12345"));
         assertThat(ListenersUtils.advertisedPortFromOverrideOrParameter(listener, 0, 12345), is("12345"));
     }
+
+    @ParallelTest
+    public void testAllocateLoadBalancerNodePorts() {
+        assertThat(internalListeners.stream()
+                        .filter(l -> ListenersUtils.allocateLoadBalancerNodePorts(l) == null ||
+                                     ListenersUtils.allocateLoadBalancerNodePorts(l))
+                        .collect(Collectors.toList()),
+                hasSize(4));
+        assertThat(allListeners.stream()
+                        .filter(l -> ListenersUtils.allocateLoadBalancerNodePorts(l) == null ||
+                                ListenersUtils.allocateLoadBalancerNodePorts(l))
+                        .collect(Collectors.toList()),
+                hasSize(13));
```

Comment:
To be honest, I have no idea what are these tests trying to do. I would just create 3 listeners here - one with the flag set to true, one with it set to false and one with it not set. And just call the method on them and assert the result.

Reply:
addressed this as well. Thanks